### ISSUE TITLE
Can't compressor minify .scss files?

### ISSUE THREAD

**Issue by SHxKM** [django-compressor/django-compressor#938]
`compressor` is minifying JS files just fine for me, but not (S)CSS. I am getting a single .CSS file, but it's never minified. I've tried all sorts of settings but none of them seem to produce a minified CSS file.

Relevant snippet from `settings.py`:

```
# SASS
COMPRESS_PRECOMPILERS = (
    ('text/x-sass', 'django_libsass.SassCompiler'),
    ('text/x-scss', 'django_libsass.SassCompiler'),
)

COMPRESS_FILTERS = {
    'css': ['compressor.filters.css_default.CssAbsoluteFilter', 'compressor.filters.cssmin.rCSSMinFilter'],
    'js': ['compressor.filters.jsmin.JSMinFilter']
}

COMPRESS_ENABLED = True
COMPRESS_OFFLINE = True
```

And in my `base.html` I have this:

```
{% compress css %}
        <link type="text/x-scss" href="{% static "stylesheets/bootstrap.scss" %}" rel="stylesheet" media="all">
{% endcompress %}
```

**Comment by jrief** [comment]
why would you like to compress .scss files anyway? They first have to be compiled to .css, otherwise your browser doesn't know what to do with that stuff. Presumably `compressor.filters.cssmin.rCSSMinFilter` doesn't know either how to handle .scss files.

If you want to include .scss files into your Django templates, you can use [django-sass-processor](https://github.com/jrief/django-sass-processor).

**Comment by diox** [comment]
I don't know anything about this problem, but using `.scss` files with compressor is a perfectly valid use case, that's what [precompilers](https://django-compressor.readthedocs.io/en/latest/settings/#django.conf.settings.COMPRESS_PRECOMPILERS) are here for.

**Comment by SHxKM** [comment]
> why would you like to compress .scss files anyway? They first have to be compiled to .css, otherwise your browser doesn't know what to do with that stuff. Presumably `compressor.filters.cssmin.rCSSMinFilter` doesn't know either how to handle .scss files.
> 
> If you want to include .scss files into your Django templates, you can use [django-sass-processor](https://github.com/jrief/django-sass-processor).

I'd like to first refer you to [the docs of django-compressor](https://django-compressor.readthedocs.io/en/stable/settings/#django.conf.settings.COMPRESS_PRECOMPILERS), section `COMPRESS_PRECOMPLIRES`.


> They first have to be compiled to .css,

They are, incidentally, by django-compressor.

> your browser doesn't know what to do with that stuff.

I knew that, I've mentioned that I'm getting a `.css` file.

**Comment by karyon** [comment]
@SHxKM your last comment came across to me as rather snarky and added little value to the discussion, please refrain from that in the future. 

regarding your report: thanks, that looks like a bug. any help in debugging this, or even better a PR, would be welcome.

**Comment by SHxKM** [comment]
@karyon you are right, I shouldn’t have replied with the same attitude I was replied to.

I’ll try to single this out and report back. Any pointers are much appreciated.

**Comment by karyon** [comment]
if you want to dive into the code, i think [this](https://github.com/django-compressor/django-compressor/blob/5fd1aed5bb0c1febaa2aa3e0fc629c450743e9f5/compressor/base.py#L231) is the line after which your css should be minified.
